
void cycle_for_catch_error(std::string, int, bool&);
int catch_error();
